src/backend/distributed/executor/local_executor.c

```diff
@@ -308,6 +317,9 @@ ExecuteLocalTaskListExtended(List *taskList,
 				totalRowsProcessed +=
 					LocallyPlanAndExecuteMultipleQueries(queryStringList, tupleDest,
 														 task);
+
```

Comment:
It is unexpected that we don't have test coverage for these lines. Should we add in this PR (maybe with prepared statements)?

Reply:
hmz, I think this is supposed to get exercised when create_distributed_table uses local execution to create shards.

Reply:
ah, it seems those commands will go through:
```c
        if (isUtilityCommand)
        {
            ExecuteUtilityCommand(TaskQueryString(task));

            MemoryContextSwitchTo(oldContext);
            MemoryContextReset(loopContext);
            continue;
        }

```

so it seems the code is unused